exp-player/addon/components/exp-player.js

```diff
@@ -142,6 +143,7 @@ able to continue the study.
         next() {
             console.log('next');
             var frameIndex = this.get('frameIndex');
+            this.sendAction('updateFrameIndex', frameIndex);
```

Comment:
Sorry if I'm misreading or missing something... but won't the call on L146 (at least in ISP) increment `frameIndex`? The it gets incremented again here on L150.